libs/core/web/src/lib/context/AuthContext.tsx

```diff
@@ -0,0 +1,29 @@
+import {
+  createContext,
+  Dispatch,
+  ReactElement,
+  SetStateAction,
+  useState,
+} from 'react';
+
+export interface IAuthContext {
+  children: ReactElement;
+}
+
+interface ContextValue {
+  isLogged: boolean;
+  setIsLogged: Dispatch<SetStateAction<boolean>>;
+}
+const defaultValue = { isLogged: false, setIsLogged: () => null };
+export const IsLoggedContext = createContext<ContextValue>(defaultValue);
+
+export function AuthContext({ children }: IAuthContext) {
+  const [isLogged, setIsLogged] = useState(false);
+  return (
+    <IsLoggedContext.Provider value={{ isLogged, setIsLogged }}>
```

Comment:
Integrate login, logout functions here and set the isLogged inside so you don't have to expose `setIsLogged`.